"mvn install" fails when using Java 8 due to stricter javadoc validation

How to reproduce:
Point JAVA_HOME to Java 8

> cd hawtio/hawtio-system
> mvn install

Possible solutions:
1) Fix the javadoc 
or
2) Turn off doclint via pom.xml (http://blog.joda.org/2014/02/turning-off-doclint-in-jdk-8-javadoc.html). Example:  `<properties><additionalparam>-Xdoclint:none</additionalparam></properties>`

Affected modules:
hawtio-dozer-schema
hawtio-json-schema-generator
hawtio-maven-indexer
hawtio-system

Further reading:
http://stackoverflow.com/questions/15886209/maven-is-not-working-in-java-8-when-javadoc-tags-are-incomplete

mvn output (truncated non-related output):
...
1 error
14 warnings
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 5.364s
[INFO] Finished at: Thu Mar 27 14:10:56 CET 2014
[INFO] Final Memory: 19M/103M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.8:jar (attach-javadocs) on project hawtio-system: MavenReportException: Error while creating archive:
[ERROR] Exit code: 1 - /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/jmx/QuartzFacadeMBean.java:12: warning: no @param for schedulerObjectName
[ERROR] void updateSimpleTrigger(String schedulerObjectName, String triggerName, String groupName, int misfireInstruction,
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/jmx/QuartzFacadeMBean.java:12: warning: no @param for triggerName
[ERROR] void updateSimpleTrigger(String schedulerObjectName, String triggerName, String groupName, int misfireInstruction,
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/jmx/QuartzFacadeMBean.java:12: warning: no @param for groupName
[ERROR] void updateSimpleTrigger(String schedulerObjectName, String triggerName, String groupName, int misfireInstruction,
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/jmx/QuartzFacadeMBean.java:12: warning: no @throws for java.lang.Exception
[ERROR] void updateSimpleTrigger(String schedulerObjectName, String triggerName, String groupName, int misfireInstruction,
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/jmx/QuartzFacadeMBean.java:21: warning: no @param for schedulerObjectName
[ERROR] void updateCronTrigger(String schedulerObjectName, String triggerName, String groupName, int misfireInstruction,
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/jmx/QuartzFacadeMBean.java:21: warning: no @param for triggerName
[ERROR] void updateCronTrigger(String schedulerObjectName, String triggerName, String groupName, int misfireInstruction,
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/jmx/QuartzFacadeMBean.java:21: warning: no @param for groupName
[ERROR] void updateCronTrigger(String schedulerObjectName, String triggerName, String groupName, int misfireInstruction,
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/jmx/QuartzFacadeMBean.java:21: warning: no @throws for java.lang.Exception
[ERROR] void updateCronTrigger(String schedulerObjectName, String triggerName, String groupName, int misfireInstruction,
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/web/AuthenticationContainerDiscovery.java:13: warning: no @return
[ERROR] String getContainerName();
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/web/AuthenticationHelpers.java:8: warning: no @param for realm
[ERROR] public static boolean isEmptyOrAllRealm(String realm) {
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/web/AuthenticationHelpers.java:8: warning: no @return
[ERROR] public static boolean isEmptyOrAllRealm(String realm) {
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/web/ProxyDetails.java:143: warning: no @param for text
[ERROR] protected int indexOf(String text, String... values) {
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/web/ProxyDetails.java:143: warning: no @param for values
[ERROR] protected int indexOf(String text, String... values) {
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/web/ProxyDetails.java:143: warning: no @return
[ERROR] protected int indexOf(String text, String... values) {
[ERROR] ^
[ERROR] /Users/user/projects/hawtio/hawtio-system/src/main/java/io/hawt/web/tomcat/TomcatAuthenticationContainerDiscovery.java:16: error: self-closing element not allowed
[ERROR] \* <p/>
[ERROR] ^
[ERROR] 
[ERROR] Command line was: /Library/Java/JavaVirtualMachines/jdk1.8.0.jdk/Contents/Home/bin/javadoc @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/Users/user/projects/hawtio/hawtio-system/target/apidocs' dir.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
...

--- a/pom.xml
+++ b/pom.xml
@@ -49,6 +49,7 @@
   </prerequisites>
 
   <properties>
+    <additionalparam>-Xdoclint:none</additionalparam>
     <jettyPort>8080</jettyPort>
     <context>/hawtio</context>
     <hawtio-config-dir>${basedir}/hawtio-config</hawtio-config-dir>

